documentation/homework/week3/todo/src/App.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import './App.css';
+import moment from 'moment'
+
+// Components
+import AssignItem from './components/Assign'
+import ListItem from './components/ListItem'
+
+// Params
+import Params from './Params/locals'
+
+class App extends Component {
+  state = {
+    list: []
```

Comment:
Can you use a more accurate/descriptive name for this array?

Reply:
Yes sure I will 👍